component/src/test/java/com/iluwatar/component/GameObjectTest.java

```diff
@@ -0,0 +1,65 @@
+package com.iluwatar.component;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.awt.event.KeyEvent;
+
+/**
+ * Tests GameObject class.
+ * src/main/java/com/iluwatar/component/GameObject.java
+ */
+class GameObjectTest {
+
+    //creates player & npc objects for testing
+    GameObject playerTest = GameObject.createPlayer();
+    GameObject npcTest = GameObject.createNpc();
```

Comment:
Maybe create these objects in @Before or @BeforeClass annotated method?

Reply:
I notice that the @Before is renamed to @BeforeEach in juint5, just about to finish all changes. 🥇